homepairs/homepairsUI/tests/redux/account/FetchAccount.test.tsx

```diff
@@ -206,51 +211,58 @@ describe('FetchAccount Action', () => {
     const mock = new MockAdapter(axios);
 
     describe('Tests Action when response returned is success', () => {
-        it('Test when the role is a PM', () => {
+        beforeEach(()=>{
+          propertyManagerMock1.clearActions();
+        });
+
+        it('Test when the role is a PM', async () => {
           const data = { 
               status: 'success',
               ...testJsonValues.PM_WITH_TOKEN,
               role: 'pm',
           };
           mock.onPost('https://homepairs-alpha.herokuapp.com/API/login/').reply(200, data);
-          propertyManagerMock1.dispatch(AccountActions.fetchAccount(email, password, testProps.navigation)).then(() => {
-            expect(spyFunction.call).toHaveLength(1);
-            const actionResults = propertyManagerMock1.getActions();
-            expect(actionResults).toHaveLength(2);
-            expect(actionResults[0]).toStrictEqual(expectedResults.PM_WIT_TOKEN);
-            expect(actionResults[1]).toStrictEqual([]);
-          });
+          await propertyManagerMock1.dispatch(
+            AccountActions.fetchAccount(email, password, testProps.navigation))
+            .then(() => {
+              expect(spyFunction.call).toHaveLength(1);
+              const actionResults = propertyManagerMock1.getActions();
+              expect(actionResults).toHaveLength(2);
+              expect(actionResults[0]).toStrictEqual(expectedResults.PM_WITH_TOKEN);
+              expect(actionResults[1]).toStrictEqual(expectedEmptyTenant);
+            });
       });
 
-      it('Test when the role is a Tenant with a property', () => {
+      it('Test when the role is a Tenant with a property', async () => {
         const data = { 
             status: 'success',
             ...testJsonValues.TENANT_WITH_TOKEN,
             role: 'tenant',
         };
         mock.onPost('https://homepairs-alpha.herokuapp.com/API/login/').reply(200, data);
-        propertyManagerMock1.dispatch(AccountActions.fetchAccount(email, password, testProps.navigation)).then(() => {
+        await propertyManagerMock1.dispatch(AccountActions.fetchAccount(email, password, testProps.navigation)).then(() => {
           expect(spyFunction.call).toHaveLength(1);
           const actionResults = propertyManagerMock1.getActions();
           expect(actionResults).toHaveLength(2);
```

Comment:
Can / should we check actionResults[0] as well (not just [1])?

Reply:
We don't actually care about the actionResults since these had. We had tested these in previous suites. We simply care that the amount of dispatch functions on a successful response has been called. No need to make redundant test (which I already have plenty of)

Reply:
" tested these in previous suites" g2k